.circleci/config.yml

```diff
@@ -18,18 +18,13 @@ jobs:
           path: ~/portal
       - run:
           name: Installing angular cli
-          command: 'npm install -g @angular/cli@6.0.8'
-      - restore_cache:
-          key: dependency-cache-{{ checksum "package.json" }}
+          command: 'npm install -g @angular/cli@6.0.8 --no-progress'
       - run:
           name: Installing npm deps
-          command: 'npm install'
-      - save_cache: # special step to save the dependency cache
-          key: dependency-cache-{{ checksum "package.json" }}
```

Comment:
Why are we removing the caching process ? @anoophm

Reply:
will revert this back. @rajeevsathish suggested this was done last time we they faced the issue. But this dint solve the issue